Implement Calculator logic for operations

AbayomiOlaoye opened the pull request:
This PR shows the implementation of MathGician's calculator logic.
> - I leveraged on the files provided for the operation: logic/calculate.js and logic/operate.js
> - I used states and props to implement nice UI responsiveness
> - All operations work correctly

AbayomiOlaoye committed:
Implement Calculator logic for operations

Wes-Isaac commented on src/components/Calculator.js:
```diff
@@ -1,52 +1,71 @@
               </td>
             </tr>
             <tr className="row">
-              <td>7</td>
-              <td>8</td>
-              <td>9</td>
-              <td className="orange">x</td>
+              <td><button type="button" onClick={this.computeFunction} name="7">7</button></td>
+              <td><button type="button" onClick={this.computeFunction} name="8">8</button></td>
+              <td><button type="button" onClick={this.computeFunction} name="9">9</button></td>
+              <td className="orange"><button type="button" onClick={this.computeFunction} name="x">x</button></td>
             </tr>
             <tr className="row">
-              <td>6</td>
-              <td>5</td>
-              <td>4</td>
-              <td className="orange">-</td>
+              <td><button type="button" onClick={this.computeFunction} name="6">6</button></td>
+              <td><button type="button" onClick={this.computeFunction} name="5">5</button></td>
+              <td><button type="button" onClick={this.computeFunction} name="4">4</button></td>
+              <td className="orange"><button type="button" onClick={this.computeFunction} name="-">-</button></td>
             </tr>
```
- [ ] Great job so far, one of the best practices is to write DRY code, it will be better to create a Button component and pass the keys to that component to keep our code tidy, it will also help as practice using props and lifecycle methods 👍.

AbayomiOlaoye replied:
Thank you, that was an oversight. It's actually an unused portion of the code. As you can see, I already created my button component.

I will effect that as soon as I can.

Wes-Isaac reviewed (changes requested):
Hi @AbayomiOlaoye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README 👍.
- use of gitflow 💯.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

AbayomiOlaoye committed:
Effect code reviewer's suggestion
Create button component to keep code DRY and appropriate the use of props for lifecycle

TedLivist commented on src/components/Calculator.js:
```diff
@@ -1,52 +1,70 @@
-              <td className="orange">x</td>
+              <Button computeFunction={this.computeFunction} name="9" />
+              <Button computeFunction={this.computeFunction} name="8" />
+              <Button computeFunction={this.computeFunction} name="7;" />
```
- [ ] You seem to have introduced an error that was not present during the previous review. I am unable to implement any operation related to this number because what you have is "7;" instead of just "7". Kindly fix that.

TedLivist reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @AbayomiOlaoye,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AbayomiOlaoye committed:
Fix bug on key <kbd>7</kbd> as detected by code reviewer

youmari reviewed (approved):
# Project Approved :trophy: :tada: 🟢 

Hi @AbayomiOlaoye 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
